motoman_driver/MotoPlus/Controller.c

```diff
@@ -227,7 +227,7 @@ BOOL Ros_Controller_Init(Controller* controller)
 		if(grpNo < controller->numGroup)
 		{
 			// Determine if specific group exists and allocate memory for it
-			controller->ctrlGroups[grpNo] = Ros_CtrlGroup_Create(grpNo, controller->interpolPeriod);
+			controller->ctrlGroups[grpNo] = Ros_CtrlGroup_Create(grpNo, (grpNo==(controller->numGroup-1)), controller->interpolPeriod);

```

Comment:
Could this line get a bit more of a comment?

It probably becomes clearer when looking up the declaration of `Ros_CtrlGroup_Create(..)`, but what is the boolean there?

Reply:
New commit has additional comments.